gigascience/gigadb-website
# A11y - home page - dataset types

# Pull request for issue: #1367

This is a pull request for the following functionalities:

![image](https://github.com/gigascience/gigadb-website/assets/143437854/f2b68545-6bd1-413c-a7df-4a32ef4ec1d7)

* Improve accessibility of "+ more" button
* Heading levels are not skipped
* Cleanup and restyle "+ more" button
* Fix low color contrast issues
* More prominent links
* Better accessibility of section under dataset types

## How to test?

Visit home page and interact with the dataset types section:
* use keyboard to navigate and check that all interactive elements are focusable and have a visible focus state
* run AXE audit (eg with chrome extension) and check that no heading levels are skipped and there are no color contrast issues

## How have functionalities been implemented?

* Links and buttons need to be noticable even when not hovered. I opted for bold darker color
* Added aria-label attributes to + more button and items in bottom section to make them more descriptive to screen readers
* Simplified link labels to avoid redundancy

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
